odl/set/domain.py

```diff
@@ -519,25 +513,17 @@ def insert(self, index, other):
         >>> rbox2 = IntervalProd([0, 0], [1, 0])
         >>> rbox.insert(1, rbox2)
         IntervalProd([-1.0, 0.0, 0.0, 2.0], [-0.5, 1.0, 0.0, 3.0])
-        >>> rbox.insert(2, [-1.0, 0.0])
-        IntervalProd([-1.0, 2.0, -1.0, 0.0], [-0.5, 3.0, -1.0, 0.0])
-
-        Can also insert by array
-
-        >>> rbox.insert(0, 1).squeeze() == rbox
-        True
-
-        See Also
-        --------
-        append
```

Comment:
I now removed the corresponding behavior also from `TensorGrid`, i.e. `insert` expects a compatible type and does not guess any longer. This inserting of lists which would then be interpreted as grids or intervals was always kind of shaky in my view, but if we are desperate to keep it I can add it back.